resources/xlf/LocProject.json

```diff
@@ -1,93 +0,0 @@
-{
-  "Projects": [
```

Comment:
undo this delete

Reply:
i just excluded this, somehow it got deleted. Let me undo it. Thanks